packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -78,11 +80,6 @@
   justify-content: flex-start;
 }
 
-.mdc-snackbar--open,
-.mdc-snackbar--closing {
-  transform: none;
```

Comment:
I'm kind of curious why this was needed to begin with, I'm assuming it's totally not needed now at any rate

Reply:
Correct. I was just paranoid about the snackbar receiving mouse events while it was closed, and I previously wasn't setting `display: none` on the root `mdc-snackbar` element, so I positioned the root offscreen using `transform: translateY(200vh)` while closed.